### ISSUE TITLE
Add all-contributors

### ISSUE THREAD

**Issue by RichardLitt** [catalyst-network/Community#4]
Right now, the CONTRIBUTING.md doc references [all-contributors](https://allcontributors.org). In order to set this up, we should enable the [all-contributors bot](https://allcontributors.org/docs/en/bot/usage). This seems the easiest way to get it up and running, rather than using a local npm package.json and having people install and run it (as suggested in the current CONTRIBUTING.md, which wouldn't work regardless.)

**Comment by nshCore** [comment]
Yes we have this running on the main repo's are you able to install it or do you need me to?
